wordpress/wp-content/themes/cds-default/header.php

```diff
@@ -101,7 +101,7 @@
 
   <?php
     $headerMenu = get_top_nav();
-    if ($headerMenu) :
+    if ($headerMenu && !cds_is_maintenance_mode()) :
```

Comment:
I don't know if we care, but there is also a custom footer menu that might show up. For consistency, we could disallow both.

<img width="702" alt="Screen Shot 2021-12-29 at 13 14 30" src="https://user-images.githubusercontent.com/2454380/147691672-31ec473c-f9a4-435b-9c39-c56eee5745cf.png">

Reply:
Updated